qjs.c

```diff
@@ -530,7 +530,7 @@ int main(int argc, char **argv)
         }
 
         for(i = 0; i < include_count; i++) {
-            if (eval_file(ctx, include_list[i], module))
+            if (eval_file(ctx, include_list[i], JS_EVAL_TYPE_GLOBAL))
```

Comment:
Does that actually work? The argument is a trinary, right?
https://github.com/quickjs-ng/quickjs/blob/caa1bf544d7cd96997abddcf31765a51fc737f16/qjs.c#L91-L98

Reply:
By chance lol, because JS_EVAL_TYPE_GLOBAL is 0. Will fix!